src/services/agent-service.ts

```diff
@@ -68,9 +68,20 @@ export default class AgentService {
   private async handleSnapshot(request: express.Request, response: express.Response) {
     profile('agentService.handleSnapshot')
 
+    // truncate domSnapshot for the logs if it's very large
+    let domSnapshotLog = request.body.domSnapshot
+    if (domSnapshotLog.length > Constants.MAX_LOG_LENGTH) {
+      domSnapshotLog = domSnapshotLog.substring(0, Constants.MAX_LOG_LENGTH)
```

Comment:
I think you'll like this one @Robdel12. We'll no longer spit out huge logs that are 99% the `domSnapshot` content.